src/prometheus.rs

```diff
@@ -5,6 +5,8 @@ use std::time::Duration;
 pub struct Config {
     pub global: GlobalConfig,
     pub scrape_configs: Vec<ScrapeConfig>,
+    #[serde(default, skip_serializing_if = "Option::is_none")]
+    pub rule_files: Option<Vec<String>>,
```

Comment:
Maybe we can make this a `Vec<String>` and then skip serializing if it is empty. That would be a easier interface to work with.